.travis.yml

```diff
@@ -3,18 +3,24 @@ language: php
 php:
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+env:
+  - COMPOSER_OPTS=""
+  - COMPOSER_OPTS="--prefer-lowest"
+
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7
   fast_finish: true
 
-sudo: false
```

Comment:
It tells travis to use the new container architecture which is way faster. Old repos don't have it enabled by default on travis, so you must include it for now if you want the container architecture.